src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Models/BankIdLoginApiErrorResponse.cs

```diff
@@ -7,6 +9,14 @@ internal BankIdLoginApiErrorResponse(string errorMessage)
             ErrorMessage = errorMessage;
         }
 
+        public BankIdLoginApiErrorResponse(string errorMessage, CollectHintCode hintCode)
+        {
+            ErrorMessage = errorMessage;
+            Retry = CollectHintCode.StartFailed.Equals(hintCode);
```

Comment:
Nice, but I would move it to the BankId Controller.So this will look like:

`public BankIdLoginApiErrorResponse(string errorMessage, bool retry)`

Reply:
Of course. Glad someone is thinking!